### PR TITLE
remove useless deprecated checking of file existence

### PR DESCRIPTION
**Pull request by matejkriz**

### CODE REVIEW

File: bootstrap-sass-styles.loader.js [hunk 1]
```diff
@@ -44,21 +44,14 @@ var partials = [
...
 var bootstrapSassPath = require('./bootstrapSassPath');
 var logger = require('./logger');
 
 function addImportReturnDependency(loader, config, propertyName) {
-  var msg;
   var fileNameResolved;
   var fileName = config[propertyName];
   if (fileName && fileName.length > 0) {
     fileNameResolved = path.relative(loader.context, fileName);
-    if (!fs.existsSync(fileNameResolved)) {
```
Comment (justin808) [inline bootstrap-sass-styles.loader.js]: @matejkriz,

The filename is configurable. It's very easy to enter the wrong name. We need some some sort of message when the file cannot be found. 

I'm guessing something changed with `fs.existsSync` that is preventing it from working. Any suggestions on how we can still get a nice message if the file doesn't exist? or does the loader.addDependency take care of such a message for us?

Comment (matejkriz) [inline bootstrap-sass-styles.loader.js]: I still get the error message:
```
File to import not found or unreadable: unexistingFileName.scss
```

Comment (matejkriz) [inline bootstrap-sass-styles.loader.js]: Just with wrong line number:

```
ERROR in ./~/css-loader!./~/sass-loader!./~/bootstrap-sass-loader/bootstrap-sass-styles.loader.js!./app/styles/bootstrap-sass.config.js
Module build failed: 
    'popover': true,
       ^
      File to import not found or unreadable: unexistingFileName.scss
      in /path/to/my/app/styles/bootstrap-sass.config.js (line 41, column 9)
 @ ./~/style-loader!./~/css-loader!./~/sass-loader!./~/bootstrap-sass-loader/bootstrap-sass-styles.loader.js!./app/styles/bootstrap-sass.config.js 4:14-368 11:1-15:3 12:19-373
```

**Comment by justin808** [comment]
Thanks @matejkriz!

**Comment by matejkriz** [comment]
Thank you @justin808!

**Pull request merged**